src/server/index.ts

```diff
@@ -23,7 +23,7 @@ const app = new koa();
 const router = new koaRouter();
 
 // Default port to listen
-const port = 8080;
```

Comment:
This entire file contains changes not related to this PR, these are already in `dev` branch

Reply:
Yeah, I know, that's an artifact of me doing a really dumb rebase earlier. I don't think it will have negative effects (shouldn't result in merge conflicts) and I cleaned up the commit history to hide the shame so I'm not sure how worth it is to fix the several issues I created.